apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -678,13 +675,6 @@ i.infoAlertIcon {
 
 // Start of rubrics redesign
```

Comment:
Do you think this comment is still needed?

Reply:
Probably not